spec/lib/tasks/after_tasks.rake_spec.rb

```diff
@@ -301,3 +301,66 @@
     expect(borked_tag.name).to eql("_\"'quotes'\"")
   end
 end
+
+describe "rake After:delete_invalid_pseud_icon_data" do
+  let!(:valid_pseud) do
+    create(
+      :pseud,
+      icon_alt_text: "hi",
+      icon_comment_text: "okay",
+      icon: File.new(Rails.root.join("public/images/feed-icon-14x14.png"))
+    )
+  end
+  let(:invalid_pseud) do
+    create(
+      :pseud,
+      icon: File.new(Rails.root.join("public/images/feed-icon-14x14.png"))
+    )
+  end
+
+  before do
+    ArchiveConfig.ICON_ALT_MAX = 5
```

Comment:
Can you stub `ArchiveConfig` instead of editing it? These changes will last after this spec.